x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview.tsx

```diff
@@ -0,0 +1,152 @@
+  canCreateInvitations: boolean;
+  isOldAccount: boolean;
+  sourcesCount: number;
+  pendingInvitationsCount: number;
+  accountsCount: number;
+  personalSourcesCount: number;
+  activityFeed: IFeedActivity[];
+  organization: {
+    name: string;
+    defaultOrgName: string;
+  };
+  isFederatedAuth: boolean;
+  currentUser: {
+    firstName: string;
+    email: string;
+    name: string;
+    color: string;
+  };
+  fpAccount: IAccount;
+}
+
+export const defaultServerData = {
+  accountsCount: 1,
+  activityFeed: [],
+  canCreateContentSources: true,
+  canCreateInvitations: true,
+  currentUser: {
+    firstName: '',
+    email: '',
+    name: '',
+    color: '',
+  },
+  fpAccount: {} as IAccount,
+  hasOrgSources: false,
+  hasUsers: false,
+  isFederatedAuth: true,
+  isOldAccount: false,
+  organization: {
+    name: '',
+    defaultOrgName: '',
+  },
+  pendingInvitationsCount: 0,
+  personalSourcesCount: 0,
+  sourcesCount: 0,
+  telemetryStatus: {},
+} as IAppServerData;
+
+const ONBOARDING_HEADER_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.overviewOnboardingHeader.title',
+  { defaultMessage: 'Get started with Workplace Search' }
+);
+
+const HEADER_TITLE = i18n.translate('xpack.enterpriseSearch.workplaceSearch.overviewHeader.title', {
+  defaultMessage: 'Organization overview',
+});
+
+const ONBOARDING_HEADER_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.overviewOnboardingHeader.description',
+  { defaultMessage: 'Complete the following to set up your organization.' }
+);
+
+const HEADER_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.overviewHeader.description',
+  { defaultMessage: "Your organizations's statistics and activity" }
+);
+
+export const Overview: React.FC = () => {
+  const { http } = useContext(KibanaContext) as IKibanaContext;
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasErrorConnecting, setHasErrorConnecting] = useState(false);
+  const [appData, setAppData] = useState(defaultServerData);
+
+  const getAppData = async () => {
+    try {
+      const response = await http.get('/api/workplace_search/overview');
+      setAppData(response);
+      setIsLoading(false);
+    } catch (error) {
+      setHasErrorConnecting(true);
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    getAppData();
+  }, []);
+
+  if (hasErrorConnecting) return <ErrorState />;
+  if (isLoading) return <Loading />;
+
+  const {
+    hasUsers,
+    hasOrgSources,
+    isOldAccount,
+    organization: { name: orgName, defaultOrgName },
+  } = appData as IAppServerData;
+  const hideOnboarding = hasUsers && hasOrgSources && isOldAccount && orgName !== defaultOrgName;
+
+  const headerTitle = hideOnboarding ? HEADER_TITLE : ONBOARDING_HEADER_TITLE;
+  const headerDescription = hideOnboarding ? HEADER_DESCRIPTION : ONBOARDING_HEADER_DESCRIPTION;
+
+  return (
+    <EuiPage restrictWidth className="overview">
```

Comment:
I think this was an old CSS class name - we can scrap it
```suggestion
    <EuiPage restrictWidth>
```

Reply:
Addressed in cdcff72